src/isar/apis/models/models.py

```diff
@@ -1,14 +1,28 @@
-from typing import Optional
+from typing import List, Optional, Union
+from uuid import UUID
 
 from pydantic import BaseModel
 
 
-class StartResponse(BaseModel):
+class StartFailedResponse(BaseModel):
     message: str
-    started: bool
-    mission_id: Optional[str] = None
 
 
-class StopResponse(BaseModel):
+class StepResponse(BaseModel):
+    id: UUID
+    type: str
+
+
+class TaskResponse(BaseModel):
+    id: UUID
+    tag_id: Optional[str]
+    steps: List[StepResponse]
+
+
+class StartMissionResponse(BaseModel):
+    id: Union[UUID, int, str, None]
```

Comment:
Why are there so many options for the mission id?

Reply:
This only reflects the possibilities given in the definition of the mission id. Not sure why there is so many different options for the id of a mission. This would be a separate issue.